.github/workflows/node-sdk.yml

```diff
@@ -0,0 +1,67 @@
+name: Whispir Node SDK generator
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  Create-Release:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Get current date
+        id: date
+        run: echo "::set-output name=date::$(date +"%Y-%m-%dT%H-%M-%S%z")"
+      - name: date
+        run: echo ${{steps.date.outputs.date}}
+      # Checkout whispir/openapi repo
+      - uses: actions/checkout@v2
+      # Clone whispir/whispir-node
+      - name: Clone Whispir-Node
+        if: success()
+        uses: whispir/checkout@main
+        with:
+          repository: whispir/${{secrets.GH_REPO}}
+          ref: "main"
+          path: whispir-node
+          token: ${{ secrets.GH_TOKEN }}
+      # Create release branch
+      - name: Create release branch
+        run: |
+          cd whispir-node/
+          git checkout -b release/${{steps.date.outputs.date}}
+      # Generate Node SDK
+      # SDK GENERATION
+      - uses: actions/setup-node@v3
+        with:
+          node-version: 16
+      - run: |
+          cd whispir-sdk-codegen/
+          yarn add @openapitools/openapi-generator-cli
+          yarn generate
+      - name: Commit changelog and manifest files
+        id: make-commit
+        run: |
+          cd whispir-node/
+          git config --global user.email ""${{GITHUB_ACTOR}}@users.noreply.github.com"
+          git config --global user.name "${{GITHUB_ACTOR}}"
+          git add -A
+          git commit --message "Prepare release ${{steps.date.outputs.date}}"
+          echo "::set-output name=commit::$(git rev-parse HEAD)"
+      - name: Push new branch
+        run: |
+          cd whispir-node/
+          git push origin release/${{steps.date.outputs.date}}
+      # Create PR
+      - name: Create pull request into main
+        uses: whispir/create-pull-request@master
```

Comment:
Is it possible to modify the whispir/create-pull-request action branch/tag from `master` to `main`. We have an obligation in diversity to avoid usage of culturally insensitive language. Link to [further reading on master vs main](https://www.theserverside.com/feature/Why-GitHub-renamed-its-master-branch-to-main).